spring-cloud-gcp-data-datastore/src/main/java/com/google/cloud/spring/data/datastore/core/LazyUtil.java

```diff
@@ -41,7 +41,7 @@
  */
 final class LazyUtil {
 
-	static private final ObjenesisStd objenesis = new ObjenesisStd();
+	static final private ObjenesisStd objenesis = new ObjenesisStd();
```

Comment:
This one should be `private static final`

Reply:
Whoops - fixed.